main/inc/lib/userportal.lib.php

```diff
@@ -1917,6 +1917,9 @@ public function returnCoursesAndSessions(
                                                     }
                                                 }
                                             }
+
+                                            $course_session['extrafields'] = CourseManager::getExtraFieldsToBePresented($course['real_id']);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 140 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5300916)